src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_frame_utils.py

```diff
@@ -35,7 +35,12 @@ def add_exception_to_frame(frame, exception_info):
 
 
 def remove_exception_from_frame(frame):
-    frame.f_locals.pop("__exception__", None)
+    # In 3.13 frame.f_locals became a proxy for a dict, so we need to copy it to avoid
```

Comment:
Grammar? Can't parse this sentence

Reply:
Sorry I didn't read this after copilot generated it. Should be fixed now.